lib/widgets/selection.dart

```diff
@@ -1160,13 +1165,9 @@ class _SelectionCounterState extends State<SelectionCounter> with SelectionHandl
       key: key,
       childKey: ValueKey(selectionCount),
       valueIncreased: controller.lengthIncreased,
-      child: Container(
-        /// Line up width with other actions, so they animate identically with [EmergeAnimation]
-        constraints: const BoxConstraints(minWidth: NFConstants.iconButtonSize),
```

Comment:
Ah, I see you explained it in the PR description. I'd prefer to keep it here, because otherwise animation looks weird. Instead, we could disable this padding when we use the StyledText

Reply:
Can you point me towards the situation where the animation looks weird? Also, are we talking about the counter animation? Why is that affected by the minimum width? I don't notice any different behavior of the counter animation with this change. If possible, I would rather resolve the weirdness for both cases, when we use StyledText and when we don't.

Reply:
`selectedTitle = false` and `counter = true`

> Also, are we talking about the counter animation?

Correct

> If possible, I would rather resolve the weirdness for both cases, when we use StyledText and when we don't.

The weirdness is only seen when `selectedTitle = false`, because it animates too short distance

Reply:
>The weirdness is only seen when selectedTitle = false, because it animates too short distance

I'm sorry, I cant seem to be able to see anything wrong, can you show me how to replicate the problem? I'm also wondering how the horizontal size would influences the animation which goes vertical.

> It can be seen when selectedTitle = false and counter = true

The only place I see this scenario is the counter in the lower left for the selection in the `artistContent` route. But I don't see a difference, expect for the padding:

| Before (`1.0.8`) | After (`feature/improve_translatabillity`) |
|-----------------|---------------------------------------------|
| ![Selection Animation Before](https://user-images.githubusercontent.com/6966049/168490778-b4abc89a-c84e-492d-858c-249d8551b284.gif) | ![Selection Animation After](https://user-images.githubusercontent.com/6966049/168490782-c96882bc-a9b5-4725-ab9b-d2d5e7dae61f.gif) |

Reply:
The weirdness is really subtile to notice and happens during EmergeAnimation, so I'm okay with removing this.